depthai_demo.py

```diff
@@ -27,6 +27,7 @@
     import depthai as dai
     import numpy as np
     from depthai_sdk.managers import ArgsManager, getMonoResolution, getRgbResolution
+    from depthai_helpers.app_manager import App
 except Exception as ex:
     print("Third party libraries failed to import: {}".format(ex))
     print("Run \"python3 install_requirements.py\" to install dependencies or visit our installation page for more details - https://docs.luxonis.com/projects/api/en/latest/install/")
```

Comment:
`or visit our installation page for more details - https://docs.luxonis.com/projects/api/en/latest/install/"` - This page is only for depthai-python install (API), not depthai-demo requirements, so I would remove that text